src/utils.ts

```diff
@@ -86,3 +86,7 @@ export function removeSelection() {
     selection.removeAllRanges();
   }
 }
+
+export function distance(x: number, y: number) {
+  return Math.abs(x > y ? x - y : y - x);
+}
```

Comment:
This is a redundant check. 
`Math.abs(x-y)` is all you need

Reply:
:blush: will commit a refactor in a bit

Reply:
But I reckon instead of that, I'll keep the expression, and remove the `Math.abs` --- for perf?

Dunno how I ended doing both.

Reply:
Don't think this sorta little perf really matters :)